README.md

```diff
@@ -1,11 +1,19 @@
 # workshop-genai-hubql
 
 
-# Repo
+### Prerequisites
+To be able to run [supabase](https://supabase.com/) please make sure you have [Docker](https://www.docker.com/products/docker-desktop/) installed on your host machine.
 
-pnpm i
+Node version: x.x.x
 
-pnpm dev
+### Local Setup
+
+```bash
+cp .env.example .env
+npm i
```

Comment:
@tobias-hubql Yeah sure we could, but it introduce one more step to setup the user's machine since `pnpm` is not the most popular package management. We already push them to get Docker. Personally, as an attendant, I wouldn't be very excited to install a bunch of seems-as-unrelated ( to gen-ai of course ) tools to my machine.

Reply:
Forgot to add, I've tried to run the project with every other tools I have ( yarn | bun | pnpm ) but only `npm` started with no troubles.